tests/precalculations/test_count_vectorizer.py

```diff
@@ -0,0 +1,51 @@
+import unittest
+from shift_detector.precalculations.count_vectorizer import CountVectorizer
+from shift_detector.precalculations.store import Store
+import pandas as pd
+import numpy as np
+
+
+class TestCountVectorizer(unittest.TestCase):
+
+    def setUp(self):
+        self.count1 = CountVectorizer(cols='col1', stop_words='english', max_features=2)
+        self.count2 = CountVectorizer(cols='col1', stop_words='english', max_features=2)
+        self.count3 = CountVectorizer(cols='col1', stop_words='english', max_features=3)
+
+        self.df1 = pd.DataFrame({'col1':
+                                ['duck', 'duck', 'duck', 'duck', 'duck',
+                                 'duck', 'duck', 'duck', 'duck', 'goose']})
+        self.df2 = pd.DataFrame({'col1':
+                                ['goose', 'goose', 'goose', 'goose', 'goose',
+                                 'goose', 'goose', 'goose', 'goose', 'duck']})
+        self.store = Store(self.df1, self.df2)
+
+    def test_eq(self):
+        self.assertTrue(self.count1 == self.count2)
+        self.assertFalse(self.count1 == self.count3)
+
+    def test_exception_for_max_features(self):
+        self.assertRaises(ValueError, lambda: CountVectorizer(cols='', max_features=0))
+        self.assertRaises(TypeError, lambda: CountVectorizer(cols='', max_features=3.5))
+
+    def test_exception_for_stop_words(self):
+        self.assertRaises(Exception, lambda: CountVectorizer(cols='', stop_words='abcd'))
+        self.assertRaises(Exception, lambda: CountVectorizer(cols='', stop_words=['english', ' abcd']))
+        self.assertRaises(TypeError, lambda: CountVectorizer(cols='', stop_words=['english', 42]))
+
+    def test_hash(self):
+        self.assertEqual(hash(self.count1), hash(self.count2))
+
+    def test_process(self):
+        res1, res2 = self.count1.process(self.store)
+        test_dict1 = {'col1': np.array([[1,0] if not i == 9 else [0,1] for i in range(10)])}
+        test_dict2 = {'col1': np.array([[0,1] if not i == 9 else [1,0] for i in range(10)])}
+
+        self.assertEqual(res1.keys(), test_dict1.keys())
+        for val1, test_val1 in zip(res1.values(), test_dict1.values()):
```

Comment:
This looks way to complicated for a test ...
Should it not be possible to just use `self.assertListEqual(test_dict1['col'], res1['col1'])`